atomicapp/display.py

```diff
@@ -0,0 +1,221 @@
+"""
+ Copyright 2015 Red Hat, Inc.
+
+ This file is part of Atomic App.
+
+ Atomic App is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Lesser General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Atomic App is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public License
+ along with Atomic App. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import os
+import sys
+import time
+import logging
+import logging.handlers
+from atomicapp.constants import LOG_NAME, LOG_LEVELS, LOG_CUSTOM_NAME, LOG_SYSLOG
+
+
+class Display:
+
+    '''
+
+    In order to effectively use logging within Python, we manipulate the level
+    codes by adding our own custom ones. By doing so, we are able to have different
+    log legs effectively span all Python files.
+
+    Upon initialization, Atomic App checks to see if --logging-ouput= is set during
+    initalization and set's the appropriate log level based on either the default
+    or custom level value.
+
+    Default python level codes
+
+    NOTSET    0
+    DEBUG    10
+    INFO     20
+    WARNING  30
+    ERROR    40
+    CRITICAL 50
+
+    Custom log levels in constants.py
+
+    LOG_LEVELS = {
+        "default": 90,
+        "cockpit": 91,
+        "stdout": 92,
+        "syslog": 93,
+        "none": 94
+    }
+
+    '''
+
+    # Console colour codes
+    codeCodes = {
+        'black': '0;30', 'bright gray': '0;37',
+        'blue': '0;34', 'white': '1;37',
+        'green': '0;32', 'bright blue': '1;34',
+        'cyan': '0;36', 'bright green': '1;32',
+        'red': '0;31', 'bright cyan': '1;36',
+        'purple': '0;35', 'bright red': '1;31',
+        'yellow': '0;33', 'bright purple': '1;35',
+        'dark gray': '1;30', 'bright yellow': '1;33',
+        'normal': '0'
+    }
+
+    # Upon initialization we grab the log level value and verbose level (-v or not)
+    def __init__(self):
+        self.logger = logging.getLogger(LOG_NAME)
+        self.verbose_level = self.logger.getEffectiveLevel()
+        self.log_level = logging.getLogger(LOG_CUSTOM_NAME).getEffectiveLevel()
+
+    def debug(self, msg, only=None):
+        self.display(msg, 10, 'cyan', only)
```

Comment:
should we be using the `DEBUG` constant here instead of `10`? Same goes for the statements below